src/WindowsPackageManager/main.cpp

```diff
@@ -54,4 +56,34 @@ extern "C"
         RETURN_HR(::Microsoft::WRL::Module<::Microsoft::WRL::ModuleType::OutOfProc>::GetModule().UnregisterObjects());
     }
     CATCH_RETURN();
+
+    WINDOWS_PACKAGE_MANAGER_API WindowsPackageManagerInProcModuleInitialize() try
+    {
+        ::Microsoft::WRL::Module<::Microsoft::WRL::ModuleType::InProc>::Create();

```

Comment:
We don't need an initialization of our code?  I suppose that might be what the state separation config would do.
We don't need to `RegisterObjects`?  Is that out-of-proc only?

Reply:
Correct. RegisterObjects is for out of proc only. In proc automatically recognizes the creator map.